src/site/filters/liquid.js

```diff
@@ -208,8 +212,16 @@ module.exports = function registerFilters() {
 
   // check is this is a root level page
   liquid.filters.isRootPage = path => {
+    const isFacilityRoot = /^(?:\/pittsburgh-health-care)+$|^(?:\/pittsburgh-health-care)\/((?!stories|events|locations|press-releases|health-services|jobs-careers).)*$/;
```

Comment:
This doesn't seem like a sustainable approach as we add more regions.

Reply:
yeah I agree. I want to check in with the content/migration team and see if there's a nailed down pattern for facility region names. If there is, this regex can be updated to be less hardcoded. I also want to try out importing paths that should be treated as root from a separate helper file and just looping through them to build this regex in a future sprint.